src/mqtt/IotaMqttService.cc

```diff
@@ -45,23 +47,80 @@ std::string iota::esp::ngsi::IotaMqttService::publishContextBroker(
   return doPublishCB(apikey, idDevice, jsonMsg);
 }
 
+void iota::esp::ngsi::IotaMqttService::publishMultiAttribute(
+    std::string& multi_payload, std::string& apikey, std::string& idDevice) {
+  std::string sep_medidas = UL20_MEASURE_SEPARATOR;
```

Comment:
medidas ... That's Spanish ...

Better to have all variable names and comments in English, one never knows who might end up maintaining this piece of code in the future ...

At least one more variable in Spanish, further down: separador